package.json

```diff
@@ -1,16 +1,19 @@
 {
   "version": "3.0.0",
   "name": "@aeternity/aepp-components",
-  "main": "build/aepp-components.js",
+  "main": "dist/aepp.components.js",
   "description": "Reusable Vue Components for aeternity aepp projects",
   "homepage": "https://github.com/aeternity/aepp-components#readme",
   "author": "Max Dörfler <max@apeunit.com> (https://github.com/muxe)",
   "license": "ISC",
   "repository": "https://github.com/aeternity/aepp-components.git",
   "bugs": "https://github.com/aeternity/aepp-components/issues",
   "files": [
+    "src",
     "dist",
-    "src"
+    "dist/aepp.base.css",
+    "dist/aepp.components.css",
+    "dist/aepp.components.js"
```

Comment:
`"dist"` does not already includes child files? why you have reordered `"src"`?

Reply:
fixed.